src/__tests__/reducers/forgot.password.reducer.test.js

```diff
@@ -0,0 +1,29 @@
+import expect from 'expect';
+import { RESET_PASSWORD_ACTION, ERROR_ACTION } from '../../constants/index';
+import restRequest from '../../reducers/reset.password.reducer';
+import RESPONSES from '../../mock/responses';
+
+const initialState = {
```

Comment:
Initial state should be in its own file i.e. `initialState.js` which will have different state objects.